src/atoms/HorizontalRule/HorizontalRule.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import HorizontalRule from './HorizontalRule';
+
+afterEach(cleanup);
+
+test('It renders an html hr element', () => {
+  const { container } = render(<HorizontalRule />);
+
+  expect(container.firstChild.tagName).toEqual('HR');
+  expect(container.innerHTML).toEqual('<hr>');
```

Comment:
nit picky but should this be ` <hr />  `?

Reply:
Which part? Also, I didn't really add this stuff haha: I just moved it around and made it use the new `react-testing-library` syntax.

Reply:
disregard I misread it